src/Security/MemberAuthenticator/LostPasswordHandler.php

```diff
@@ -173,7 +177,18 @@ public function forgotPassword(array $data, Form $form): HTTPResponse
         if ($member) {
             $token = $member->generateAutologinTokenAndStoreHash();
 
-            $this->sendEmail($member, $token);
+            $success = $this->sendEmail($member, $token);
+            if (!$success) {
+                $form->sessionMessage(
+                    _t(
+                        Member::class . '.EMAIL_FAILED',
```

Comment:
Need to add this to en.yml

Reply:
Will do - though it'll fall back to that until it's added, and tx-translator will add it when run, so in future I think we don't need to worry about that during PRs.

Reply:
Done